apps/console/src/features/applications/components/edit-application.tsx

```diff
@@ -167,6 +167,8 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
 
     const eventPublisher: EventPublisher = EventPublisher.getInstance();
 
+    const isFragmentApp = application.advancedConfigurations.fragment;
```

Comment:
```suggestion
const isFragmentApp: boolean = application.advancedConfigurations.fragment;
```

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3617/commits/a1ce3c88d00366850288b9dc6c28a27368bcf14e